src/screens/AccountRecover.js

```diff
@@ -31,8 +36,9 @@ import TextInput from '../components/TextInput';
 import { NETWORK_LIST } from '../constants';
 import AccountsStore from '../stores/AccountsStore';
 import { validateSeed } from '../util/account';
-import { debounce } from '../util/debounce'
+import { debounce } from '../util/debounce';
 import { brainWalletAddress } from '../util/native';
+import KeyboardScrollView from '../components/KeyboardScrollView';
```

Comment:
(nit) alphabetical order, because the import is from `../components/KeyboardScrollView`, it should be placed after `./components/Button`